### PR TITLE
Remove legacy HTTP and websocket

### PR DESCRIPTION
**Pull request by andersjohnsen**
Should we check in .packages? Would probably be much easier for the users.

### CODE REVIEW

File: tests/http_test.toit [hunk 1]
```diff
@@ -1,249 +0,0 @@
-// Copyright (C) 2018 Toitware ApS. All rights reserved.
```
Comment (kasperl) [inline tests/http_test.toit:1]: Do we have a test like this for the new http package?

Comment (andersjohnsen) [reply tests/http_test.toit:1]: There are examples, yes

Comment (kasperl) [reply tests/http_test.toit:1]: I worry that we get more and more untested functionality in packages and we will have no way of knowing if we're breaking things. Can we add tests to the packages (in this case the http one) and set it up so we run our package tests through GitHub actions?

Comment (andersjohnsen) [reply tests/http_test.toit:1]: Sure, but sounds a bit out of scope for this change.

Comment (kasperl) [reply tests/http_test.toit:1]: I can see that - and I am all for getting this cleaned up for users of the Toit SDK. Will you file an issue for the postponed testing, so we don't forget?

File: examples/README.md [hunk 2]
```diff
@@ -2,6 +2,15 @@
 
 The Toit examples are meant to get you started writing code for an ESP32.
 
+## Packages
+
+Some of the examples use other packages, e.g. HTTP. To run the examples you first need to run
```
Comment (kasperl) [inline examples/README.md:7]: Nit: Add ':' after run for consistency with other READMEs.

File: examples/README.md [hunk 3]
```diff
@@ -2,6 +2,15 @@
...
+
+```
```
Comment (kasperl) [inline examples/README.md:9]: Drop $ from the instructions and change it to:

``` sh
build/host/sdk/bin/toitpkg pkg install
```

(adding sh and using the built toitpkg)

**Review by kasperl — APPROVED** [review]
Looks good to me with comments.

**Pull request merged**